developer/src/kmc/test/test-project-build.ts

```diff
@@ -4,11 +4,13 @@ import 'mocha';
 import { BuildProject } from '../src/commands/buildClasses/BuildProject.js';
 import { makePathToFixture } from './helpers/index.js';
 import { InfrastructureMessages } from '../src/messages/infrastructureMessages.js';
+import { clearOptions } from '@keymanapp/developer-utils';
 
 const callbacks = new TestCompilerCallbacks();
 
 describe('BuildProject', function () {
   it('should build a keyboard project', async function() {
+    clearOptions();
```

Comment:
Would this be consistent with the other test files?

```suggestion
  beforeEach(clearOptions);
  
  it('should build a keyboard project', async function() {
```